chart/docker-images.yaml

```diff
@@ -2,14 +2,14 @@
   "dockerImage": {
     "reverseProxy": "docker.io/nginx:1.20",
     "jobs": {
-      "mongodbMigration": "huggingface/datasets-server-jobs-mongodb_migration:sha-5364f81"
+      "mongodbMigration": "huggingface/datasets-server-jobs-mongodb_migration:sha-690d1cd"
     },
     "services": {
-      "admin": "huggingface/datasets-server-services-admin:sha-5364f81",
-      "api": "huggingface/datasets-server-services-api:sha-5364f81"
+      "admin": "huggingface/datasets-server-services-admin:sha-4dc0383",
+      "api": "huggingface/datasets-server-services-api:sha-4dc0383"
     },
     "workers": {
-      "datasets_based": "huggingface/datasets-server-workers-datasets_based:sha-5364f81"
+      "datasets_based": "huggingface/datasets-server-workers-datasets_based:sha-690d1cd"
```

Comment:
This is to update which docker image to deploy right ?
I understand you have to update admin, api and datasets_based (you did changes to all of them) but why do you update mongodbMigration ?